.github/workflows/unit.yml

```diff
@@ -39,7 +39,7 @@ jobs:
         run: |
           cargo nextest run \
             --locked --features "asm-keccak ${{ matrix.network }}" \
-            --workspace --exclude examples --exclude ef-tests \
+            --workspace --exclude ef-tests \
```

Comment:
same

Reply:
we have some tests in examples (these tests are also kind of examples of how to write tests), so i would prefer running them